tests/test_vpc.py

```diff
@@ -2492,6 +2492,65 @@ def test_endpoint_cross_account(self):
         self.assertEqual(violations[0]['Effect'], 'Allow')
 
 
+class InternetGatewayTest(BaseTest):
+
+    def test_query_internet_gateways(self):
```

Comment:
Not sure we would need these tests. There's already a test here: https://github.com/cloud-custodian/cloud-custodian/blob/e21823f0b94dd23bc9bbac88424e5be2256acfd9/tests/test_query.py#L146

Reply:
Oh I didn't see that one. Should I keep the tagging test or only the delete one?

Reply:
Imo, just a test around your feature(action: delete) should be good enough